### DISCUSSION TITLE
The swordsman x survival

### DISCUSSION THREAD

**Discussion by IamWildDew** [gameserverapp/Platform#417] [Game request]
id like to see this game be able to be hosted by you guys as i love your stuff!!!! It launches 4-27-23
https://store.steampowered.com/app/1588560/The_Swordsmen_X_Survival/

**Comment by mvaessen** [comment]
Hi, I can't find any info about this game supporting private servers. Do you have any info where this is mentioned?

**Reply by IamWildDew** [reply]
it been mentioned in the official discord and community discord from the people that have been translating from the devs for us. They just got a new publisher so i believe the official discord will be moving. But time will tell. The following is all the info we have at the moment im hoping we get publishers/devs giving us more info soon. The release date was just announced wednesday so it may be a few days before we get more info. I can get you both discords if needed
![image](https://user-images.githubusercontent.com/28834479/232233965-8e91981c-496c-4daa-b76a-8794f95300b9.png)

**Reply by mvaessen** [reply]
Hi, feel free to add updates when they become available.
